backend/application/docker/docker_management_service.py

```
@@ -0,0 +1,400 @@
+            # Check Docker daemon connectivity
+            docker_ping = await self._host_repository.ping()
+            if not docker_ping:
+                return {
+                    'valid': False,
+                    'error': 'Cannot connect to Docker daemon'
+                }
+            
+            # Check data directories
+            data_dirs = stack.get_all_data_directories()
+            missing_dirs = []
+            for data_dir in data_dirs:
```

Ruff (0.11.9)

Loop control variable `data_dir` not used within loop body

Rename unused `data_dir` to `_data_dir`

(B007)